terraform/20-app/etc/auth-dev.tfvars

```diff
@@ -0,0 +1,2 @@
+environment_type = "auth-dev"
+halo_account_type = "nl7"
```

Comment:
These were confirmed by the vendors as being the other way around in the end.
So `auth-dev` should map to `nl8`
and `auth-test` should map to `nl7`

Reply:
Ahh thanks, have swapped them around